core/src/test/java/nl/nn/adapterframework/align/TestMap2Xml.java

```diff
@@ -87,85 +88,85 @@ public void testNestedValue() throws Exception {
 
 	@Override
 	@Test
-	@Ignore("Map2XML does not support mixed content")
+	@Disabled //("Map2XML does not support mixed content")
```

Comment:
Waarom heb je hier de comments uit de annotatie gehaald?
```suggestion
	@Disabled("Map2XML does not support mixed content")
```

Reply:
ah, wist niet dat dat kon. Pas ik even aan